controller.py

```diff
@@ -54,4 +56,88 @@ def run(quad, des_state):
                    k_p_theta * (des_theta - theta) + k_d_theta * (q_des - q),
                    k_p_psi * (des_psi - psi) + k_d_psi * (r_des - r)]]).T
 
+    print(F)
+    print(M)
     return F, M
+
+def run_LQR(quad, des_state):
+
+    # get drone state
+    x, y, z = quad.position()
+    x_dot, y_dot, z_dot = quad.velocity()
+    phi, theta, psi = quad.attitude()
+    p, q, r = quad.omega()
+
+    # get desired state
+    des_x, des_y, des_z = des_state.pos
+    des_x_dot, des_y_dot, des_z_dot = des_state.vel
+    des_x_ddot, des_y_ddot, des_z_ddot = des_state.acc
+    des_psi = des_state.yaw
+    des_psi_dot = des_state.yawdot    
```

Comment:
it does not look like these are used, please review the input parameters, and only takes the one that is used.